low-level/sensors/impl/generic/node_sas_port.py

```diff
@@ -462,6 +453,11 @@ def _create_json_message(self, alert_type, port):
                     }
         else:
             # This is a port level alert
+            if alert_type == 'fault':
+                description = f"No connectivity detected on the SAS port {port}, possible \
```

Comment:
\# For SAS HBA level connection alert
Fault 'Description' : "SAS connection error detected in SAS HBA %s " % self.RESOURCE_ID
Fault_Resolved 'Description' : "SAS connection re-established in SAS HBA %s " % self.RESOURCE_ID

Reply:
added description ofr SAS HAB alerts.